librz/bin/pdb/symbol.c

```diff
@@ -25,7 +22,7 @@ RZ_IPI bool PDBSectionOffset_parse(RzBuffer *b, PDBSectionOffset *section_offset
 
 static bool symbol_name_parse(RzBuffer *b, PDBSymbolKind kind, char **result) {
 	if (kind < S_ST_MAX) {
-		// TODO: pascal
+		return buf_read_u8_pascal_string(b, result);
```

Comment:
Do we have a file with this case?

Reply:
I have not found PDB files with this case.